indy_common/state/domain.py

```diff
@@ -75,13 +75,17 @@ def prepare_attr_for_state(txn):
     """
     assert txn[TXN_TYPE] in {ATTRIB, GET_ATTR}
     nym = txn[TARGET_NYM]
-    attr_key, value = parse_attr_txn(txn)
-    hashed_value = hash_of(value) if value else ''
+    attr_type, attr_key, value = parse_attr_txn(txn)
+    if attr_type == ENC:
```

Comment:
Can't we just remove 
```
    if attr_type == ENC:
        return hash_of(attr), attr
```
from `parse_attr_txn`?

Reply:
Agreed, done.